### PR TITLE
Docs: fix FOUC in dark theme

### PR DESCRIPTION
**Pull request by ycw**
before https://raw.githack.com/mrdoob/three.js/r149/docs/index.html#manual/en/introduction/Creating-a-scene

https://user-images.githubusercontent.com/1063018/218715339-fe8ded97-10a5-47bd-8103-4a72c6af2025.mp4

---

after https://raw.githack.com/ycw/three.js/FOUC/docs/index.html#manual/en/introduction/Creating-a-scene

https://user-images.githubusercontent.com/1063018/218715413-ca321b00-5908-4df0-8745-8a123c875cb2.mp4

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -575,6 +577,18 @@ <h1><a href="https://threejs.org">three.js</a></h1>
 
 		//
 
+		window.addEventListener( 'message', (ev) => {
```
[the remark this answers is not in the document]
Comment (ycw) [reply docs/index.html]: good!

**Comment by Mugen87** [comment]
Can you please explain a bit how the new code solves the FOUC issue.

**Comment by mrdoob** [comment]
Most importantly, can you share a screen recording of that FOUC?

**Comment by Mugen87** [comment]
While I'm at video recording.....this is how it looks on latest Chrome on macOS.

https://user-images.githubusercontent.com/12612165/218702047-e83d3c51-836b-4e5d-a986-1f12f17dd884.mov

The screen does not flicker with this PR.

**Comment by Mugen87** [comment]
@ycw Thanks for the additional explanations.

In rare cases (about 1 out of 7), I still see the flash even with your version after some subsequent reloads.

**Comment by ycw** [comment]
@Mugen87 updated to await prettyprint() **and** page.css, could you test again, thanks.

**Comment by Mugen87** [comment]
Unfortunately, the last commit does not fix the issue for me. Still getting occasional flashes:

**Comment by ycw** [comment]
unresolved: `color-scheme: light dark` causes 'flash of white background' in iframe

**Comment by Mugen87** [comment]
Thanks for giving it a try!

**Pull request closed without merging**